deployment/dobi.yaml

```diff
@@ -40,16 +40,21 @@ job=deploy-eventhubdb:
              kubectl create namespace $TENANT --dry-run=client -o yaml | kubectl apply -f - &&
              kubectl delete secret ci4rail-harbor-secret --namespace=$TENANT --ignore-not-found &&
              kubectl create secret docker-registry ci4rail-harbor-secret \
-                                                   --namespace=$TENANT \
-                                                   --docker-server=$DOCKER_REGISTRY \
-                                                   --docker-username=$DOCKER_REGISTRY_USERNAME \
-                                                   --docker-password=$DOCKER_REGISTRY_PASSWORD \
-                                                   --validate=true &&
+                                    --namespace=$TENANT \
```

Comment:
This is true for all variables that get replaced by sed:
Please rename `FOO` to `TEMPLATE_VAR_FOO`, e.g. `TENANT` -> `TEMPLATE_VAR_TENANT`